packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -4773,31 +4777,36 @@ QUnit.module('Aria accessibility', {
         }).dxCalendar('instance');
 
         let $cell = $(getCurrentViewInstance(calendar).$element().find(toSelector(CALENDAR_SELECTED_DATE_CLASS)));
-        assert.equal($cell.attr('aria-selected'), 'true', 'aria-selected was added to the main view cell');
+        assert.strictEqual($cell.attr('aria-selected'), 'true', 'aria-selected is true on the main view cell');
 
         $cell = $(getAdditionalViewInstance(calendar).$element().find(toSelector(CALENDAR_SELECTED_DATE_CLASS)));
-        assert.equal($cell.attr('aria-selected'), 'true', 'aria-selected was added to the additional view cell');
+        assert.strictEqual($cell.attr('aria-selected'), 'true', 'aria-selected is true on the additional view cell');
     });
 
     ['multiple', 'range'].forEach((selectionMode) => {
         QUnit.test(`aria-selected on selected date cell, selectionMode=${selectionMode}`, function(assert) {
             const calendar = this.$element.dxCalendar({
                 value: [new Date(2015, 5, 1)],
                 selectionMode,
-                focusStateEnabled: true
+                focusStateEnabled: true,
             }).dxCalendar('instance');
 
             const keyboard = keyboardMock($(calendar._$viewsWrapper));
 
-            let $cell = $(getCurrentViewInstance(calendar).$element().find(toSelector(CALENDAR_SELECTED_DATE_CLASS)));
-            assert.equal($cell.attr('aria-selected'), 'true', 'aria-selected was added to the cell');
+            let $cell = $(getCurrentViewInstance(calendar).$element().find(`.${CALENDAR_CELL_CLASS}:not(.${CALENDAR_SELECTED_DATE_CLASS})`)[1]);
```

Comment:
[minor old copypaste]
cannot we reduce copypaste by storing view element to a variable?

Reply:
[6b7af7397147d040d7d393367e922d42f8bdf33c]